sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -419,6 +423,10 @@ case class DescribeTableCommand(table: TableIdentifier, isExtended: Boolean, isF
     val result = new ArrayBuffer[Row]
     val catalog = sparkSession.sessionState.catalog
 
+    if (partitionSpec.nonEmpty) {
+      catalog.getPartition(table, partitionSpec)
```

Comment:
Does this touch the result? How is the partition information added to the result?